builder/azure/arm/config_retriever.go

```diff
@@ -1,5 +1,12 @@
 package arm
 
+// Method to resolve information about the user so that a client can be
+// constructed to communicated with Azure.
```

Comment:
hmmm, seems that this is also up for a rename... in some future refactor... :wink: